crates/mongodb-connector/src/api_type_conversions/query_request.rs

```diff
@@ -105,6 +105,7 @@ fn find_object_field<'a>(
         ConversionError::UnknownObjectTypeField {
             object_type: object_type.name.to_string(),
             field_name: field_name.to_string(),
+            path: Default::default(), // TODO: set a path for more helpful error reporting
```

Comment:
Is this TODO for later or missed in this PR?

Reply:
Saving this for later